cmd/aws-iam-authenticator/server.go

```diff
@@ -83,31 +83,31 @@ func init() {
 
 	serverCmd.Flags().String("generate-kubeconfig",
 		"/etc/kubernetes/aws-iam-authenticator/kubeconfig.yaml",
-		"Output `path` where a generated webhook kubeconfig (for `--authentication-token-webhook-config-file`) will be stored (should be a hostPath mount).")
+		"Output `path` where a generated webhook kubeconfig (for `--authentication-token-webhook-config-file`) will be stored.  When running as a container, this should probably be a hostPath mount.")
```

Comment:
why? this is assuming that apiserver is running on the same host and so can access this kubeconfig right? I would state that assumption, the point is to make the webhook readable by apiserver,  to further improve the comment